src/App.tsx

```diff
@@ -35,6 +36,25 @@ export function getReorderedGoods(
   const visibleGoods = [...goods];
 
   // Sort and reverse goods if needed
+
+  visibleGoods.sort((a, b) => {
+    switch (sortType) {
+      case SortType.ALPHABET:
+        return a.localeCompare(b);
+
+      case SortType.LENGTH:
+        return a.length - b.length;
+
+      case SortType.NONE:
+      default:
+        throw new Error('Unexpected sortBy value');
```

Comment:
SortType.None is expected value, in this case you just shouldn't sort array, so you could return 0 , error should throw if we pass value that doesn't exist in enum